tests/DependencyInjection/SonataAdminExtensionTest.php

```diff
@@ -17,6 +17,25 @@
 
 class SonataAdminExtensionTest extends AbstractExtensionTestCase
 {
+    /**
+     * @var string[]
+     */
+    private $defaultStylesheets = [];
+
+    /**
+     * @var string[]
+     */
+    private $defaultJavascripts = [];
+
+    protected function setUp()
+    {
+        parent::setUp();
+        $this->container->setParameter('kernel.bundles', []);
+        $this->load();
+        $this->defaultStylesheets = $this->container->getDefinition('sonata.admin.pool')->getArgument(3)['stylesheets'];
```

Comment:
It seems that his line is over 120 characters, could you please break it? Thank you